src/borg/archiver/help_cmd.py

```diff
@@ -180,6 +178,25 @@ class HelpMixIn:
         exclude stops examination of subdirectories so that potential includes
         will not match - use normal excludes for such use cases.
 
+        Example::
+
+            # Define the root path prefix
```

Comment:
maybe rather say **recursion root* (not prefix).

just to make clear that this is not precisely the prefix seen in paths going into the matcher (removal of left slash or other stuff).